cf_units/__init__.py

```diff
@@ -111,7 +111,7 @@
 #
 # cftime constants
 #
-CALENDAR_STANDARD = 'standard'
+CALENDAR_STANDARD = 'gregorian'
```

Comment:
I think that its unwise to change this here, as it's then use later in `CALENDARS` and the docs do publicise `cf_units.CALENDARS` in several places as the list of valid calendars... and `standard` is still valid, but aliased to `gregorian`.

Could we rather introduce a calendar alias lookup, and also deal with `noleap` and `all_leap` at the same time? i.e.

```python
CALENDAR_ALIAS = dict(CALENDAR_STANDARD=CALENDAR_GREGORIAN,
                      CALENDAR_NO_LEAP=CALENDAR_365_DAY,
                      CALENDAR_ALL_LEAP=CALENDAR_366_DAY)
```
Then use the `CALENDAR_ALIAS` later on to do the aliasing... ?

Reply:
Good idea. 

I'll follow it through.